src/tests/ftest/util/command_utils_base.py

```diff
@@ -347,7 +347,9 @@ def __str__(self):
         params = []
         for name in self.get_str_param_names():
             value = str(getattr(self, name))
-            if value != "":
+            # Makito modified
+            #if value != "":
+            if value != "" and value != "True" and value != "False":
```

Comment:
(pylint-consider-using-in) Consider merging these comparisons with "in" to "value not in ('', 'True', 'False')"